tests/helper/tests/reset_env_vars.py

```diff
@@ -1,62 +1,6 @@
-import logging
+def reset_env_vars(client):
+    """Test for checking if sudoers defaults set env_reset"""
+    (exit_code, _, _) = client.execute_command(
+        f"grep '^Defaults.env_reset' /etc/sudoers", quiet=True)
```

Comment:
I'd keep this on hold until #892 got merged and use a more Pythonic way (by using https://github.com/gardenlinux/gardenlinux/pull/892/files#diff-5ed4b018bb02161b990ad7d516a52d2695a738ed9968a11b86730edac180e888 `[key_val_in_file.py]`).